graylog2-server/src/main/java/org/graylog2/streams/StreamRouterEngine.java

```diff
@@ -180,7 +216,7 @@ public String getFingerprint() {
 
             for (final StreamRule streamRule : stream.getStreamRules()) {
                 try {
-                    final Rule rule = new Rule(stream, streamRule);
+                    final Rule rule = new Rule(stream, streamRule, null);
```

Comment:
Actually I will now replace it with the matching type of the stream. The wording was annoying from the beginning and a boolean flag is just not able to properly represent what it's supposed to do.